src/Services/DiscussionServiceOrig.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Firefly\Services;
+
+use Firefly\Models\Discussion;
+use Firefly\Models\Group;
+use Illuminate\Http\Request;
+
+class DiscussionService
```

Comment:
What is this file for?

Reply:
Sorry, was using this to make sure I had restored all the formatting correctly, removed in the latest push